tools/webhook_test.py

```diff
@@ -126,6 +126,9 @@ def set_init(webhook_type):
             "message": {
                 "gym_id": 0,
                 "gym_name": "unknown",
+                "team": 0,
```

Comment:
We need to leave `team` out until we get better support added for it - otherwise it could effect the value in gym cache. 

I think we should just drop all changes to the webhook tester for now, so that they will just show up as missing info (unless you have a better suggestion)